test_app/README.md

```diff
@@ -52,26 +52,24 @@ python3 -m iwa_http_server.py
 2. Execute Chrome with the following flags once.
 
 ```sh
-$CHROME --enable-features=IsolatedWebApps,IsolatedWebAppDevMode,IwaControlledFrame  \
-        --install-isolated-web-app-from-url=http://localhost:8000
+$CHROME --enable-features=IsolatedWebApps,IsolatedWebAppDevMode
 ```
 
 **Optional:** Use the `--user-data-dir` flag to install the IWA in a separate
 Chrome profile. For example:
 
 ```sh
 $CHROME --user-data-dir=$HOME/tmp \
-        --enable-features=IsolatedWebApps,IsolatedWebAppDevMode,IwaControlledFrame \
-        --install-isolated-web-app-from-url=http://localhost:8000
+        --enable-features=IsolatedWebApps,IsolatedWebAppDevMode \
```

Comment:
The backslash here is no longer necessary since the --install-isolated-web-app-from-url flag is removed.

Reply:
Done.